src/content/api/idempotency.mdoc

```diff
@@ -0,0 +1,10 @@
+---
+title: Idempotency
+description: Introduction to Idempotency
+nav:
+  path: API Reference
+  order: 9
+---
+Several key APIs support Idempotency. Idempotency allows requests to be safely retried without risk of performing the operation twice. When calling an API, provide an idempotency key. Then, if a connection error occurs, you can safely repeat the request using the same key without risk of the operation being performed twice.
```

Comment:
Nice start @laurencemcknight 💪

Some initial thoughts:
- We can probably remove the first line. "key APIs" doesn't mean anything to readers of this guide.

> When calling an API, provide an idempotency key.
- "APIs that support idempotency will have an `idempotencyKey` property."

- Clients should also repeat the request if a 500 occurs, not just a **network** error
- We've repeated the "risk of the operation being performed twice" line twice. Do you think we could reduce duplication here?
- Perhaps there could be a sequence diagram on this page to illustrate the example better? Maybe a client making 2 calls to the pay payment request route with the same idempotencyKey.
- Have you researched any examples in the industry that we could refer to?

Reply:
Love the feedback. I've addressed the first four points.
As we discussed, re the last two points:
1. We won't add a sequence diagram as we felt is was overkill
2. Yes, Stripe.

Reply:
Awesome, the guide is much clearer now!